README.md

```diff
@@ -1,24 +1,195 @@
+</details>
+<details>
+  <summary>Database</summary>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built using Ruby on Rails with React**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby >= 3.2.0
+    rails >= 7.0
+    postgres >- 15.3
+    node
+    Yarn
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/Graciano1997/hello_world_rails_react.git
+```
+
+You need to setup database for these project
+
+```
+  development = your_dataabase_dev
+  test        = your_datasebase_test
+  production  = your_datasebase_prod
+```
+
+or you can use your own database and change the ```config/database.yml```
+
+```yml
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: [your_username]
+    password: [your_password]
+    host: localhost
+
+  development:
+    <<: *default
+    database: [your_database_for_development]
+
+  test:
+    <<: *default
+    database: [your_database_for_test]
+
+  production:
+    <<: *default
+    database: [your_database_for_production]
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd hello-rails-react
+  bundle install
+  rails db:reset
+  npm install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+to use this project:
+
+```bash
+   npm run build
+   bin/rails server
+```
+
+it will run the the server on ```localhost:3000```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+👤 **GracianoHenrique**
+
+- GitHub: [@GracianoHenrique](https://github.com/Graciano1997)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add UI System**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Graciano1997/hello_world_rails_react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can share this project to your friend
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The MIT license link is broken, Please provide a valid link to make your README file professional.

![image](https://github.com/Graciano1997/hello_world_rails_react/assets/68796232/a4161458-8326-4d0a-bbf8-731dace78575)